.babelrc

```diff
@@ -35,6 +35,12 @@
       "plugins": [
         "./build/use-lodash-es"
       ]
+    },
+    "rollup": {
+      "plugins": [
+        "./build/use-lodash-es",
+        "external-helpers"
```

Comment:
With this rollup-plugin-babel calls `buildExternalHelpers` of `babel-core` and adds them only once to bundle. Not external package.